app/views/devise/passwords/edit.html.erb

```diff
@@ -18,8 +24,8 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Change my password" %>
+    <%= f.submit "Change my password", class: "btn btn-lg btn-primary btn-block" %>
```

Comment:
For now, let's go with capitalized. It can always be changed later in the translation files.